src/common/Journald.cc

```diff
@@ -107,7 +107,7 @@ class EntryEncoder : public EntryEncoderBase {
   void encode(const Entry& e, const SubsystemMap *s)
   {
     meta_buf.clear();
-    fmt::format_to(meta_buf,
+    fmt::format_to(std::back_inserter(meta_buf),
```

Comment:
might want to `#include <iterator>` explicitly in this file instead of relying some random header file to do this for us.